releasenotes/notes/cqm-constraint-data-c8217056d499f800.yaml

```diff
@@ -0,0 +1,7 @@
+---
+features:
+  - |
+    Add ``ConstrainedQuadraticModel.iter_constraint_data()``,
+    ``ConstrainedQuadraticModel.iter_violations()`` and
+    ``ConstrainedQuadraticModel.violations()`` methods.
+    See `971 <https://github.com/dwavesystems/dimod/issues/971>`_.
```

Comment:
Do we need a new blank line in the end?

Reply:
Nope! It's a good habit for code files since it means appending to file won't affect the git history for unrelated lines. But since these files are never touched again it doesn't matter. Good eye though!